index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>

```

Comment:
- [ ] As stated the project requirement kindly make sure that you added A .css file that includes `one class` with styles for your header.
![Capture](https://user-images.githubusercontent.com/93598189/204540709-42bc6aff-5d1a-40e0-9418-5b08f5d372a1.PNG)

Reply:
okey

Reply:
done